changes/16857.md

```diff
@@ -0,0 +1,14 @@
+# PR 16831: Move plonk_wasm into proof-systems
```

Comment:
This is wrong title. Can you update, please?

Nit: Having the CHANGELOG entry in a different commit would be preferred.

Reply:
Ugh sorry, copy/paste error. Still a bit rusty coming back...